src/ui/lib/api.ts

```diff
@@ -3,33 +3,17 @@ import type { ZarfState } from './api-types';
 const BASE_URL = 'api';
 const MAGIC = 'MAGIC';
 
-const Cluster = {
-	getState: request<ZarfState>('cluster/state'),
-	updateState: async (body: ZarfState) => request<ZarfState>('cluster/state', 'PUT', body)
+const headers = {
+	Authorization: `Bearer ${MAGIC}`,
+	'Content-Type': 'application/json'
 };
 
-const Packages = {};
-
-export { Cluster, Packages };
-
-async function request<Response>(
-	url: string,
-	method: string = 'GET',
-	body?: any
-): Promise<Response> {
-	try {
-		const response = await fetch(`/${BASE_URL}/${url}`, {
-			method,
-			headers: {
-				Authorization: `Bearer ${MAGIC}`,
-				'Content-Type': 'application/json'
-			},
-			body
+const getClusterState = async (): Promise<ZarfState> =>
```

Comment:
Going to play with this some more, not a fan of this rewrite of this file and it will lead to a lot of needless boilerplate.  This function wasn't even getting called from svelte, so this are doesn't really need to be rewritten.